README.md

```diff
@@ -176,18 +176,29 @@ search for a Python interpreter matching that version in the following order:
 
 ### Installing into arbitrary Python environments
 
-Since uv has no dependency on Python, it can even install into virtual environments other than
+Since uv has no dependency on Python, it can install into virtual environments other than
 its own. For example, setting `VIRTUAL_ENV=/path/to/venv` will cause uv to install into
-`/path/to/venv`, no matter where uv is installed.
-
-uv can also install into arbitrary, even non-virtual environments by providing a `--python` argument
-to `uv pip sync` or `uv pip install`. For example, `uv pip install --python=/path/to/python` will
-install into the environment linked to the `/path/to/python` interpreter.
-
-For convenience, `uv pip install --system` will install into the system Python environment, as an
-approximate shorthand for, e.g., `uv pip install --python=$(which python3)`. Though we generally
-recommend the use of virtual environments for dependency management, `--system` is intended to
-enable the use of uv in continuous integration and containerized environments.
+`/path/to/venv`, regardless of where uv is installed. Note that if `VIRTUAL_ENV` is set to
+a directory that is **not** a [PEP 405 compliant](https://peps.python.org/pep-0405/#specification)
+virtual environment, it will be ignored.
+
+uv can also install into arbitrary, even non-virtual environments, with the `--python` argument
```

Comment:
This is confusing because `--system` is _required_ for this now, right?

Reply:
No, if it's explicit (i.e. that path to the executable or its directory) we do not require the `--system` flag.